flocker/node/_loop.py

```diff
@@ -597,8 +604,16 @@ def error(failure):
         d.addErrback(error)
 
         # We're done with the iteration:
-        d.addCallback(
-            lambda delay: self.fsm.receive(delay))
+        def send_delay_to_fsm(delay):
+            Message.log(
+                message_type=u'flocker:node:_loop:CONVERGE:delay',
+                log_level=u'INFO',
+                message=u'Delaying until next convergence loop.',
+                delay=delay
+            )
+            return self.fsm.receive(delay)
```

Comment:
It says that it's sleeping, but not for how long :( Which means I can't tell if it is intentionally in an exponential back-off situation or not.